qiskit_metal/analyses/quantization/lumped_oscillator_model.py

```diff
@@ -142,14 +142,15 @@ def run_lom(self):
 
         s = self.setup
 
-        if not self.sim.capacitance_matrix:
-            self.logger.warning(
-                'Please initialize the capacitance_matrix before executing this method.'
-                '`self.sim.capacitance_matrix = pd.DataFrame(...)`')
-            return
-        if not self.sim.capacitance_all_passes:
-            self.sim.capacitance_all_passes[
-                1] = self.sim.capacitance_matrix.values
+        if isinstance(self.sim, Dict):
+            if self.sim.capacitance_matrix == {}:
```

Comment:
if self.sim is a **dictionary**, then self.sim.capacitance_matrix would never work right?

Reply:
should we check instead "`if 'capacitance_matrix' not in self.sim" or not isinstance(self.sim['capacitance_matrix'], pd.DataFrame)`"?

Reply:
understood